src/app/[locale]/dataset-list.tsx

```diff
@@ -6,6 +6,19 @@ import {SearchResult} from '@/lib/dataset-fetcher';
 import DatasetCard from './dataset-card';
 import FilterSet from './filter-set';
 import {clientSearchDatasets} from './client-search-dataset';
+import Paginator from './paginator';
+import {
+  PageSidebar,
+  PageContent,
+  PageTitle,
+  PageHeader,
+} from '@/components/page';
+
+export enum Sort {
+  RelevanceAsc = 'relevanceAsc',
```

Comment:
Perhaps intentional, but just in case: if you sort by relevance in ascending order, the least relevant results are returned first. If you want to have the most relevant results first, you have to sort in descending order.

Reply:
Was not intentional. I will change it.